packages/webpack/mixins/stats/mixin.core.js

```diff
@@ -43,7 +48,18 @@ class WebpackStatsMixin extends Mixin {
 }
 
 WebpackStatsMixin.strategies = {
-  getBuildStats: callable,
+  getBuildStats: validate(
+    callable,
+    ({ length }) => {
+      invariant(length === 0, 'getBuildStats(): Received obsolete argument(s)');
```

Comment:
Here again we have an unclear message - can we somehow make this more descriptive?